src/Component/Metadata/Operation.php

```diff
@@ -13,13 +13,13 @@
 
 namespace Sylius\Component\Resource\Metadata;
 
-class Operation
+/**
+ * @experimental
+ */
+abstract class Operation
 {
     public function __construct(
         protected ?string $name = null,
-        protected ?array $methods = null,
-        protected ?string $path = null,
-        protected ?string $routePrefix = null,
         protected ?string $controller = null,
         protected ?string $template = null,
```

Comment:
I'm wondering if template should be part of operations in general

Reply:
Yeah indeed...